resources/entrypoints/fix_user.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -e
+
+echo "[Entrypoint] RonDB Docker Image"
+
+echo "\$@: $@"
+
+# In the case we're running on Linux and the user executing build_run_docker.sh
+# is not 1000:1000, we will fail writing files to the host file system. We thereby
+# add the mysql user to the group of host user.
+
+# We need to be in the same group as the host to be able to create files
+whoami
+if [ $(getent group $HOST_GROUP_ID) ]; then
+	echo "group $HOST_GROUP_ID exists."
+else
+	echo "group $HOST_GROUP_ID does not exist."
+	groupadd -g $HOST_GROUP_ID host_group_dummy
+fi
+
+usermod -g $HOST_GROUP_ID mysql  # Overwrite primary group
+usermod -a -G mysql mysql  # Append secondary group
+echo "groups mysql: $(groups mysql)"
+
+ls -la ./docker_entrypoints/rondb_standalone
+chmod +x ./docker_entrypoints/rondb_standalone/main.sh
```

Comment:
Not sure exactly what you mean, but `chmod +x` could/should probably be run from the Dockerfile I guess.

Reply:
The Dockerfile copies these files from the repository to the image: `COPY --chown=mysql:mysql ./resources/entrypoints ./docker_entrypoints/rondb_standalone`

If the executable bit is set in the repo, it will be set in the container too. Since git keeps track of the executable bit, it seems simpler to just set it in the repo.

Reply:
Ah yes okay. Wasn't aware the permissions get copied into the image.

Reply:
I've fixed this, let's see if I can add to an existing PR...